core-bundle/src/Security/Authentication/Token/TokenChecker.php

```diff
@@ -172,4 +198,39 @@ private function getTokenFromSession(string $sessionKey): ?TokenInterface
 
         return $token;
     }
+
+    private function validatePreviewLink(FrontendPreviewToken $token): bool
```

Comment:
This method should probably be called `isValidPreviewLink`.

Reply:
```suggestion
    private function isValidPreviewLink(FrontendPreviewToken $token): bool
```